docs/pages/machine-id/reference/configuration.mdx

```diff
@@ -75,10 +75,12 @@ onboarding:
   # Support values include:
```

Comment:
We have at least two more
- ec2 which I think is not recommended (in favor of IAM)
- spacelift

Is spacelift not supported here?